.gitignore

```diff
@@ -87,6 +87,3 @@ challenge_data/*
 
 # Media Files
 media/
-
-# .DS_Store for OSX
-*/.DS_Store
```

Comment:
Why are we removing the lines 90, 91 and 92 here? These should not be the part of this PR, right?